src/Blocks/BlockCrops.h

```diff
@@ -46,33 +47,49 @@ class cBlockCropsHandler:
 
 		// Fully grown, drop the crop's produce:
 		cItems res;
+		// Get current fortune level
+		int fortune = 0;
+		if (a_Tool)
+		{
+			fortune = a_Tool->m_Enchantments.GetLevel(cEnchantments::eEnchantment::enchFortune);
+		}
+
 		switch (m_BlockType)
 		{
 			case E_BLOCK_BEETROOTS:
 			{
-				char SeedCount = 1 + ((rand.RandInt<char>(2) + rand.RandInt<char>(2)) / 2);  // [1 .. 3] with high preference of 2
+				// TODO Check Fortune Level
```

Comment:
What do you mean by check fortune level? The values here match the wiki, so the comment should either be removed or you should do any additional checks you need to do.

I don't want to introduce more TODOs where it's avoidable like this.